general-superstaq/general_superstaq/typing.py

```diff
@@ -16,3 +18,248 @@
         "state_vector": Optional[str],
     },
 )
+
+TARGET_LIST = {
```

Comment:
Yeah I wasn't entirely sure if it warranted a new file, but I've moved it to `gss.testing.py`

Reply:
another option that just occurred to me - if we do move `get_targets` to `gss.Service`, (afaict) all the testing would just have to live in `gss` so it might make more sense to put this in `gss/conftest.py`